doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-legacy-EXPECT-proc.adoc

```diff
@@ -25,6 +25,9 @@ The list includes the following options, depending on the data in the *GIVEN* se
 image::project-data/test-scenario-field-value.png[Modify a fact field]
 +
 . Set the field values to what is expected to be valid as a result of the *GIVEN* input (such as `approved` | `equals` | `false`).
++
+NOTE: In the legacy test scenarios designer, you can use the `=["value1", "value2"]` string format in the *EXPECT* field to validate the list of strings.
```

Comment:
I would suggest removing `the` ... you can use `=["value1", "value2"]` string format...

Reply:
@hmanwani-rh, I will remove "the". Thanks.